control-plane/tests/test_service_persistence_restore_readiness.py

```
@@ -3669,6 +3669,65 @@ def test_service_phase21_readiness_surfaces_source_and_automation_health(
             "reconciliation_timeout",
         )
 
+    def test_service_phase21_readiness_surfaces_optional_extension_operability_defaults(
+        self,
+    ) -> None:
+        store, _ = make_store()
```

Ruff (0.15.10)

[error] Undefined name `make_store`

(F821)